.github/workflows/docs.yml

```diff
@@ -54,8 +54,8 @@ jobs:
       - name: Build docs
         env:
           # needs to be in sync with [package.metadata.docs.rs]
-          RUSTDOCFLAGS:  -Zunstable-options --cfg=docsrs
-        run: cargo doc --all-features --no-deps -p bevy -Zunstable-options -Zrustdoc-scrape-examples
+          RUSTDOCFLAGS:  -Zunstable-options --cfg=docsrs --generate-link-to-definition
+        run: cargo doc --all-features --no-deps --workspace -Zunstable-options -Zrustdoc-scrape-examples
```

Comment:
I don't think that should change?

Reply:
On docs.rs all the crates are individually built and then an option is passed to cross link their source code (I think it should be `extern-html-root-url`). Building as a workspace does a kinda similar thing, except there's no need for the `extern-html-root-url` since all the crates documentation are together.

For comparison, before this PR only the `bevy` crate documentation is built, resulting in no source links other than those to the `src/lib.rs` file (i.e. only http://dev-docs.bevyengine.org/src/bevy/lib.rs.html). This not only is different from docs.rs, but also makes the effects of `--generate-link-to-definition` basically invisible, since it contains almost no source code where that feature has an effect.